src/test/groovy/ee/tuleva/onboarding/payment/PaymentIntegrationSpec.groovy

```diff
@@ -115,9 +135,9 @@ class PaymentIntegrationSpec extends Specification {
     SecurityContextHolder.clearContext()
     expectThatPaymentCallbackRedirectsUser()
     expectThatPaymentCallbackCreatedOnePayment(aUser)
+    mockSecurityContext()
     mockEpisTransactionsForPayment()
     expectToBeAbleToReceivePaymentNotification()
```

Comment:
In production, the payment notification incoming request will have no security context.